File: VB/FilterDropDown_AgregateOperators/Models/ClassName.vb
Imports DevExpress.Mvvm

Namespace FilterDropDown_AgregateOperators

    Public Class ClassName
        Inherits BindableBase

        Protected _Name As String

        Public Property Name As String
            Get
                Return _Name
            End Get

            Set(ByVal value As String)
                SetProperty(_Name, value, "Name")
            End Set
        End Property

        Protected _Age As Integer

        Public Property Age As Integer
            Get
                Return _Age
            End Get

            Set(ByVal value As Integer)
                SetProperty(_Age, value, "Age")
            End Set
        End Property

        Protected _Genres As Object

        Public Property Genres As Object
            Get
                Return _Genres
            End Get

            Set(ByVal value As Object)
                SetProperty(_Genres, value, "Genres")
            End Set
        End Property

        Protected _DateTime As Date

        Public Property DateTime As Date
            Get
                Return _DateTime
            End Get

            Set(ByVal value As Date)
                SetProperty(_DateTime, value, "DateTime")
            End Set
        End Property

        Protected _IsSelected As Boolean

        Public Property IsSelected As Boolean
            Get
                Return _IsSelected
            End Get

            Set(ByVal value As Boolean)
                SetProperty(_IsSelected, value, "IsSelected")
            End Set
        End Property
    End Class
End Namespace

File: CS/FilterDropDown_AgregateOperators/Models/ClassName.cs
using DevExpress.Mvvm;
using System;

namespace FilterDropDown_AgregateOperators {
    public class ClassName : BindableBase {
        protected string _Name;
        public string Name {
            get { return this._Name; }
            set { this.SetProperty(ref this._Name, value, "Name"); }
        }

        protected int _Age;
        public int Age {
            get { return this._Age; }
            set { this.SetProperty(ref this._Age, value, "Age"); }
        }

        protected object _Genres;
        public object Genres {
            get { return this._Genres; }
            set { this.SetProperty(ref this._Genres, value, "Genres"); }
        }

        protected DateTime _DateTime;
        public DateTime DateTime {
            get { return this._DateTime; }
            set { this.SetProperty(ref this._DateTime, value, "DateTime"); }
        }

        protected bool _IsSelected;
        public bool IsSelected {
            get { return this._IsSelected; }
            set { this.SetProperty(ref this._IsSelected, value, "IsSelected"); }
        }
    }
}
